netuitive/client.py

```diff
@@ -51,6 +51,7 @@ def __init__(self, url='https://api.app.netuitive.com/ingest',
         self.post_error_count = 0
         self.max_post_errors = 10
         self.connection_timeout = connection_timeout
+        self.request_repeat_time = 4
```

Comment:
My only tiny comment is I initially read this as the time between retries, not the max retry count. I recommend renaming this to `max_check_retry_count`, changing it to 3, and changing 229 to work with the change to 3.